graphql_vuestorefront/models/product.py

```diff
@@ -7,11 +7,23 @@
 from odoo import models, fields, api, tools, _
 from odoo.addons.http_routing.models.ir_http import slug, slugify
 from odoo.exceptions import ValidationError
+from odoo.tools.sql import column_exists, create_column
 
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
+    def _auto_init(self):
+        """Skip computation of the field on module installation
+        because it will fail as a result of large amount of products"""
+        if not column_exists(
+            self.env.cr, "product_template", "website_slug"
+        ):
+            create_column(
+                self.env.cr, "product_template", "website_slug", "varchar"
```

Comment:
You can add `SQL` here, to set same value as compute. This should not fail as its much more efficient than using compute.

Reply:
I would prefer if possible to not spend more time on this at the moment, will just run the compute method as a script to recalculate the values once is installed on the server.